Project.toml

```diff
@@ -17,9 +16,8 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 HSL = "0.1.0, 0.2.0"
-Krylov = "0.6, 0.7.0"
+Krylov = "0.6, 0.7"
```

Comment:
```suggestion
Krylov = "^0.7.3"
```

Reply:
Why? I just tested it, and it still works for Krylov.jl 0.6, 0.7.1, 0.7.2.

Reply:
If it still works with previous versions of Krylov.jl, let's keep them. :)